src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -233,6 +233,8 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
 
 	<AndroidSdkBuildToolsVersion Condition="'$(AndroidSdkBuildToolsVersion)' == ''">23.0.0</AndroidSdkBuildToolsVersion>
+	<AndroidSdkPlatformToolsVersion Condition="'$(AndroidSdkPlatformToolsVersion)' == ''">25.0.5</AndroidSdkPlatformToolsVersion>
+	<AndroidSdkBuildToolsVersion Condition="'$(AndroidSdkBuildToolsVersion)' == ''">25.2.5</AndroidSdkBuildToolsVersion>
```

Comment:
This property is also specified two lines above.

This PR should *alter* the existing value, not "replace" it two lines later. :-)